micronaut-jms-activemq/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation project(':micronaut-jms-core')
+    implementation platform('io.micronaut:micronaut-bom:2.0.0')
+
+    // lombok config must come before the Micronaut annotation processor config
+    compileOnly 'org.projectlombok:lombok:1.18.12'
+    annotationProcessor 'org.projectlombok:lombok:1.18.12'
+
+    implementation "javax.jms:javax.jms-api:2.0.1"
+    implementation "org.apache.activemq:activemq-broker:5.15.13"
+    annotationProcessor "io.micronaut:micronaut-inject-java"
+    // needed because otherwise IntelliJ integration broken
+    compileOnly "io.micronaut:micronaut-inject-java"
+    compile "io.micronaut:micronaut-inject"
+
+    testCompile project(':micronaut-jms-core-test')
+
+    testCompileOnly 'org.projectlombok:lombok:1.18.12'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
+
+    testCompileOnly "io.micronaut:micronaut-inject-java"
+
+    testAnnotationProcessor "io.micronaut:micronaut-inject-java"
```

Comment:
@alvarosanchez strangely I tried removing the `testAnnotationProcessor 'io.micronaut:micronaut-inject-java` and then the Jupiter engine was unable to run the tests. Is there some alternate configuration I need to do?

Reply:
If you use 2.0.10 of our Gradle plugin, as in https://github.com/micronaut-projects/micronaut-project-template/blob/master/build.gradle#L9, it should work

Reply:
Unfortunately it is still not working, I think because [this line](https://github.com/micronaut-projects/micronaut-build/blob/master/src/main/groovy/io/micronaut/build/MicronautBuildCommonPlugin.groovy#L62) adds `micronaut-inject-groovy` as a `testAnnotationProcessor` but not `micronaut-inject-java` unless I'm missing something. I have to confess I am fairly new to Gradle having pretty much exclusively used Maven in the past. I can easily rewrite the tests in Spock and get Groovy support for free if you'd prefer the Micronaut libraries to be standardized on Groovy for testing?

Reply:
Did you add this too? https://github.com/micronaut-projects/micronaut-project-template/blob/master/xxx/build.gradle#L13-L15

Reply:
I hadn't but that didn't seem to help either. I'll push up your suggested changes to see if you can see anything that I have not configured correctly.